pydantic_ai_slim/pydantic_ai/models/gemini.py

```diff
@@ -693,7 +693,7 @@ def simplify(self) -> dict[str, Any]:
 
     def _simplify(self, schema: dict[str, Any], refs_stack: tuple[str, ...]) -> None:
         schema.pop('title', None)
-        default = schema.pop('default', _utils.UNSET)
+        schema.pop('default', None)
```

Comment:
I'm not sure why we are currently requiring the default to be `None` in order to remove the the `{'type': 'null'}` from the schema, that condition doesn't seem to me to be necessary, so I removed it here. (And we don't use the value of the default at all, so I just dropped it.)